paddle/framework/op_desc.cc

```diff
@@ -52,6 +52,22 @@ class CompileTimeInferShapeContext : public InferShapeContext {
   const std::vector<std::string> &Outputs(
       const std::string &name) const override;
 
+  void ShareLoD(const std::string &in, const std::string &out, size_t i = 0,
```

Comment:
Should we rename it ReuseLoD?

Reply:
The purpose of this interface is passing the Lod info from some input to some output. Most Op will just pass it through, some op will process this Lod Information. So I think ShareLod is properly.